src/script/main/app.js

```diff
@@ -90,15 +90,16 @@ class App {
 
     this.service = this._setupServices();
     this.repository = this._setupRepositories();
-    this.util = this._setup_utils();
+    if (z.config.FEATURE.ENABLE_DEBUG) {
+      this.util = {debug: new DebugUtil(this.repository)};
```

Comment:
Would be even nicer to have `is.debug = new DebugUtil` but it requires QA to change a few stuff on their end. Will discuss this with @mythsunwind on Monday